appveyor-coverage.yml

```diff
@@ -23,6 +23,7 @@ clone_depth: 1
 install:
   - set PATH=C:\Ruby22\bin;C:\\Python34;C:\\Python34\\Scripts;%PATH%"
   - pip install codecov
+  - ps: cinst nunit-extension-vs-project-loader --no-progress

```

Comment:
inspired by https://stackoverflow.com/questions/66636785/nunit3-console-invalid-csproj-file-type-is-not-supported
Not sure if it will fix the problem, but let's try